tests/crypto/test_cases/test_aead.c

```diff
@@ -334,7 +334,7 @@ void exec_test_case_aead(void)
 {
 	int err_code = -1;
 
-#if defined(CONFIG_MBEDTLS_CCM_C)
+#if defined(CONFIG_CC310_MBEDTLS_CCM_C) || defined(CONFIG_VANILLA_MBEDTLS_CCM_C) || defined(CONFIG_MBEDTLS_CCM_C)
```

Comment:
Since we are in source code, it would suffice with MBEDTLS_CCM_C (defined in nrf-config.h)

Reply:
8abda7e09d0c871e53353e762735ecb9426cd792